src/gobexport/exporter/config/uva2.py

```diff
@@ -1191,25 +1266,57 @@ def _add_verblijfsobjecten_uva2_config():
             'VBOMNT/TijdvakRelatie/einddatumRelatie': '',
             'VBOTGG/TGG/Code': 'toegang.[0].code',
             'VBOTGG/TijdvakRelatie/begindatumRelatie': {
-                'action': 'format',
-                'formatter': format_uva2_date,
-                'value': 'beginGeldigheid',
+                'condition': 'isempty',
+                'reference': 'toegang.[0].code',
+                'falseval': {
+                    'action': 'format',
+                    'formatter': format_uva2_date,
+                    'value': 'beginGeldigheid',
+                },
+                'trueval': {
+                    'action': 'literal',
+                    'value': ''
```

Comment:
Deze code wordt vaak herhaald. Zou je er misschien niet beter een action van kunnen maken met een value. Dan is de code verbonden aan de action in plaats van elke keer herhaald

Reply:
Added a function to remove duplicate code